src/test/java/uk/gov/register/core/PostgresRegisterTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.register.core;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.register.configuration.RegisterFieldsConfiguration;
+import uk.gov.register.configuration.RegisterNameConfiguration;
+import uk.gov.register.exceptions.NoSuchFieldException;
+import uk.gov.register.store.BackingStoreDriver;
+import uk.gov.register.store.postgres.PostgresDriverTestBase;
+
+import java.util.ArrayList;
+
+import static org.mockito.Mockito.*;
+
+public class PostgresRegisterTest extends PostgresDriverTestBase {
+    private RegisterNameConfiguration registerNameConfiguration;
```

Comment:
This does not need to extend PostgresDriverTestBase as these tests do not need the mocked functionality in that class.